### PR TITLE
Feature accelerometerhandling

### PR DESCRIPTION
**Pull request by LMiljak**
Quite a lot in this pull request:
- Information from accelerometer received by the server gets processed (platform actually moves).
- Server now starts immediately when the application is started.
- Android now immediately connects to a LAN server if it has been connected.
- Many bugs with the AutoConnector fixed.

### CODE REVIEW

File: ContextDesktop/src/main/java/com/github/migi_1/Context/AccelerometerMessageHandler.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	 * Storing this instance somewhere is useless, as it gets registered
+	 * automatically. So all that needs to be done is "new AccelerometerMessageHandler();"
+	 * and you're all set.
+	 */
+	@SuppressWarnings("unchecked")
+	public AccelerometerMessageHandler(Main main) {
+		ServerWrapper.getInstance().getServer().addMessageListener(this);
+		this.main = main;
+	}
+
```
Comment (NilsHullegien) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/AccelerometerMessageHandler.java:27]: CheckStyle: Expected @param tag for  'main'

File: ContextDesktop/src/main/java/com/github/migi_1/Context/Main.java [hunk 2]
```diff
@@ -245,6 +263,10 @@ private void addListeners(InputManager inputManager, ActionListener acl) {
     }
 
 
+    public void handleAccelerometerMessage(float orientation){
```
Comment (NilsHullegien) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/Main.java]: Missing javadoc
CheckStyle: '{' is not preceded with whitespace

**Comment by NilsHullegien** [comment]
Please fix the checkstyle errors, besides that the code looks good

**Comment by LMiljak** [comment]
Fixed the CheckStyle errors.

**Comment by NilsHullegien** [comment]
Looks good now, I approve.

**Comment by dVoorhout** [comment]
everything looks fine

**Pull request merged**